src/gui/modals/FileDialog.cpp

```diff
@@ -66,6 +89,22 @@ FileDialog::FileDialog( QWidget *parent, const QString &caption,
 		urls << QUrl::fromLocalFile( volumesDir.absolutePath() );
 #endif
 
+#ifdef LMMS_BUILD_LINUX
+
+	// FileSystem types : https://www.javatpoint.com/linux-file-system
+	QStringList usableFileSystems = {"ext", "ext2", "ext3", "ext4", "jfs", "reiserfs", "ntfs3", "fuse.sshfs", "fuseblk"};
```

Comment:
Maybe I'm just a Windows brainlet, but why would you have a filesystem mounted that wasn't useable? I'm averse to whitelisting (or blacklisting) external things in the source code, as it's a burden we have to keep up-to-date, and our releases aren't exactly frequent. Is there actually any risk in displaying file systems whose types aren't in this list?

Reply:
I did it only to prevent to show volumes with system filesystems like 'tmpfs'.

Reply:
Makes sense. Is tmpfs only used for `/tmp` and its ilk, or could it reasonably be seen in normal usage too? Would it be better to blacklist tmpfs rather than whitelisting the others? Should particular system directories, rather than filesystem types, be omitted instead? I don't know what's best here, so I'll leave it up to you Linux people.

Reply:
A modern Linux has many technical file systems/types that are mounted while running. Here's a list of the types that are currently mounted on my system:
```
$ findmnt -lo fstype | sort | uniq
autofs
bpf
cgroup2
configfs
debugfs
devpts
devtmpfs
efivarfs
ext4
fusectl
hugetlbfs
mqueue
ntfs3
proc
pstore
securityfs
sysfs
tmpfs
tracefs
vfat
```

Reply:
I see. I'll resolve this conversation then, since it doesn't look like there's a better approach for now.